src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+  totalBooks: 0,
+  isLoading: true,
+};
+
+const bookSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books = action.payload;
+    },
```

Comment:
[OPTIONAL]
While adding a book, it would be nicer to assign an `id` on it as best practice. like so:
```jsx
state.books = [
        ...state.books,
        {
          id: Math.floor(Math.random() * 100),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
```